Cargo.toml

```diff
@@ -18,9 +18,12 @@ lazy_static = "~1"
 sha2 = "~0.10"
 sha3 = "~0.10"
 thiserror = "~1"
+# Clippy fix: disable "oldtime" feature for chrono
```

Comment:
Does this pass audit with the 'oldtime' removal?

Reply:
I am hoping it does. When chrono is included with stock features it doesn't iirc.

Reply:
Yes, that's correct
```
$ cargo audit && cargo clippy 
    Fetching advisory database from `https://github.com/RustSec/advisory-db.git`
      Loaded 509 security advisories (from /home/dimm/.cargo/advisory-db)
    Updating crates.io index
    Scanning Cargo.lock for vulnerabilities (220 crate dependencies)
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
```

Reply:
🎉